specification/contosowidgetmanager/Contoso.WidgetManager/main.tsp

```diff
@@ -9,21 +9,34 @@ using TypeSpec.Rest;
 using TypeSpec.Versioning;
 using Azure.Core;
 
+@useAuth(OAuth2Auth<[ContosoAuthFlow]>)
```

Comment:
@raych1: Do you think this is the best form of auth to use in the sample?  @tjprescott recommended this fix instead as a reaction to his new linter rule coming in typespec-next:

```
@useAuth(AadOauth2Auth<["https://contoso.azure.com/.default"]>)
```

https://github.com/Azure/azure-rest-api-specs/pull/24628/files#diff-367a7f65ef36d833ded6fe3ba09a1d2232837b92eb652cb9964e7508f31f3274R12

Reply:
@tjprescott recommended we switch to his auth code when we migrate to typespec-next